test/b2gedmntuples_cfg.py

```diff
@@ -275,18 +258,29 @@
 ### ------------------------------------------------------------------
 from JMEAnalysis.JetToolbox.jetToolbox_cff import jetToolbox
 
+listBtagDiscriminators = [ 'pfTrackCountingHighEffBJetTags',
+		'pfTrackCountingHighPurBJetTags',
+		'pfJetProbabilityBJetTags',
+		'pfJetBProbabilityBJetTags',
+		'pfSimpleSecondaryVertexHighEffBJetTags',
+		'pfSimpleSecondaryVertexHighPurBJetTags',
+		'pfCombinedSecondaryVertexV2BJetTags',
+		'pfCombinedInclusiveSecondaryVertexV2BJetTags',
+		'pfBoostedDoubleSecondaryVertexAK8BJetTags'
+		]
+
```

Comment:
Could be a bad lxplus day for me, but we can may be keep for now. But can you modify the PR adding the CvsL and CvsB taggers?